build.gradle

```diff
@@ -53,3 +53,46 @@ publishing {
 tasks.withType(JavaCompile) {
     options.encoding = 'UTF-8'
 }
+
+def originalFile = null;
+
+task backupCode {
+    def file = file('src/main/java/org/json/JSONParserConfiguration.java')
+    originalFile = file.text
+}
+
+task firstTest {

```

Comment:
I think 'gradlew build' will execute the unit tests, so firstTest can probably be omitted.